content/v2.0/backup-restore/backup.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Back up data
+seotitle: Back up data in InfluxDB
+description: >
+  Use the `influx backup` command to back up all data and metadata stored in InfluxDB.
+menu:
+  v2_0:
+    parent: Back up & restore data
+weight: 101
+related:
+  - /v2.0/backup-restore/restore/
+  - /v2.0/reference/cli/influx/backup/
+draft: true
+---
+
+Use the [`influx backup` command](/v2.0/reference/cli/influx/backup/) to back up
+all data and metadata stored in InfluxDB.
+InfluxDB copies all data and metadata to a file set stored in a specified location
```

Comment:
Maybe "~file set~ set of files"

Reply:
Ah I see below we use the term "file set" in the `restore` doc. Is this a technical term? Can we change it?